### PR TITLE
US2_1T2 image compression

### PR DESCRIPTION
**Pull request by FaiiFaiipuipui**
- create image compression function.

### CODE REVIEW

**Review by pccherrii — APPROVED** [review]
Great work! I've tried uploading 30 MB image and It works fine!

File: frontend/src/app/payment/edit/page.tsx [hunk 1]
```diff
@@ -91,6 +95,73 @@ export default function PaymentPage() {
...
+        reader.onloadend = () => {
+          // setImagePreview(reader.result);
+          setImagePreview(resizedFileBase64);
+        };
+
+        //Show Image Preview
+        if (resizedFileBase64) {
+          // Convert base64 string to Blob
+          const byteCharacters = atob(resizedFileBase64.split(',')[1]);
+          const byteNumbers = new Array(byteCharacters.length);
```
Comment (whourmm) [inline frontend/src/app/payment/edit/page.tsx:141]: lgtm,

File: frontend/src/app/payment/edit/page.tsx [hunk 2]
```diff
@@ -171,7 +243,64 @@ export default function PaymentPage() {
             Upload new receipt
           </div>
           <div className="mx-10 mb-20">
-            <UploadSlip />
+            <div className="">
+              <div className="">
+                {imagePreview ? (
+                  <div className="flex items-center justify-center">
+                    <Image
+                      src={imagePreview}
```
Comment (whourmm) [inline frontend/src/app/payment/edit/page.tsx:251]: LGTM, 

The reason it didn't work recently is that I forgot to include the useState() return to the parent component.
My apologies for the oversight.

Comment (Yiwahpsp) [reply frontend/src/app/payment/edit/page.tsx:251]: Oh its ok, ค่อยแก้รอบหน้าที่มีโอกาส5555555

**Pull request merged**